src/endpoints/cart.js

```diff
@@ -36,8 +36,11 @@ class CartEndpoint extends BaseExtend {
     return this.call
   }
 
-  AddProduct(productId, quantity = 1, data = {}) {
-    const body = buildCartItemData(productId, quantity)
+  AddProduct(productIdentity, quantity = 1, data = {}) {
+    const isProductId = productIdentity.match(
+      '^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$'
```

Comment:
This regex is very strict. Just want to ensure we tested this to be certain it's applied to our generated UUIDs only

Reply:
@shaunmaharaj productIds are v4 and I think this regex targets v4 only 👍

Reply:
@yasiloghmani I think the regex needs to be formatted like `/^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/i`